tools/tf2openapi/types/tfmetagraph.go

```diff
@@ -6,31 +6,28 @@ It is the internal model representation for the MetaGraph defined in the TensorF
 [tensorflow/core/protobuf/meta_graph.proto]
 */
 import (
+	"fmt"
 	"github.com/getkin/kin-openapi/openapi3"
 	pb "github.com/kubeflow/kfserving/tools/tf2openapi/generated/protobuf"
 )
 
-const PredictReqSigDefMethod string = "tensorflow/serving/predict"
-
 type TFMetaGraph struct {
-	SignatureDefs [] TFSignatureDef
```

Comment:
Should we keep this as an array? We can limit this on the ToOpenAPI side.

Reply:
changed back to slice so that the internal model fully captures the impt fields of the savedmodel